### PR TITLE
I18n improvements

### PR DESCRIPTION
**Pull request by LiquidPL**
* configured frontend libraries (date-fns, vue-timeago, flatpickr) to respect the user locale preferences,
* internationalized some more strings,
* added a Polish translation.

### CODE REVIEW

**Comment by LiquidPL** [comment]
Now that I'm thinking about it, we should also add parameter replacing to the Vue `$trans` helper for parity with Laravel's, since I've seen a few strings that could make use of it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ All notable changes to `twill` will be documented in this file.
 - Rename default migrations to prevent conflicts in existing codebases [`74173acd`](https://github.com/area17/twill/commit/74173acd0b750243c2d3bab382ed842f36efdce9)
 
 #### Frontend build
-- *@antonin and @ifox*
```
Comment (yanhao-li) [inline CHANGELOG.md:46]: Let's remove this commit `c35a20346e240415d9589fceb14fa7df9165d6c6` from the pull request.

File: lang/en/lang.php [hunk 2]
```diff
@@ -63,25 +117,39 @@
             'create-another' => 'Create and add another',
         ],
         'update' => [
-            'title' => 'Update',
+            'title' => 'Update item',
```
Comment (yanhao-li) [inline lang/en/lang.php:120]: I think we may prefer a concise name, and keep it clean and simple as `Update`

File: lang/en/lang.php [hunk 3]
```diff
@@ -145,6 +257,16 @@
     ],
     'publisher' => [
         'switcher-title' => 'Status',
+        'languages' => 'Languages',
+        'langages-published' => 'Live',
```
Comment (yanhao-li) [inline lang/en/lang.php:261]: typo here `langages` => `languages`

File: frontend/js/utils/locale.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    'date-fns': require('date-fns/locale/fr'),
+    flatpickr: French
+  },
+  pl: {
+    'date-fns': require('date-fns/locale/pl'),
+    flatpickr: Polish
+  }
+}
+
+const twelveHourFormatLocales = ['en']
```
Comment (yanhao-li) [inline frontend/js/utils/locale.js:28]: This - I guess its currently fine, but I guess in the future we should add another prop to the `_date_picker` for user to override.

File: frontend/js/components/modals/ModalCreate.vue [hunk 5]
```diff
@@ -53,7 +53,7 @@
         return this.createMode ? this.formCreate : this.action
       },
       modalTitle: function () {
-        return this.createMode ? this.$trans('modal.create.title', 'Add new') : this.$trans('modal.update.title', 'Update')
+        return this.createMode ? this.$trans('modal.create.title', 'Add new') : this.$trans('modal.update.title', 'Update item')
```
Comment (yanhao-li) [inline frontend/js/components/modals/ModalCreate.vue:56]: let's keep the label of `Add new` and `Update` consistent.

**Comment by yanhao-li** [comment]
Hi @LiquidPL ,

Since I'm working on a new approach which will impact the way how we contribute translations in the future, I have included your commits in my PR: #624

Let me know if you have any comments.

Thanks for your contribution!

**Pull request closed without merging**